Updates to "Umbraco-Cloud/Upgrades/Moving-From-Courier-To-Deploy" (#1758)
umbraco/UmbracoDocs: Umbraco-Cloud/Upgrades/Moving-From-Courier-To-Deploy/index.md

In normal circumstances, the move from Courier to Deploy will happen automatically when you click the button in the portal for upgrading to Umbraco 7.6. 
If for some reason your Umbraco project was upgraded, but not switched to Deploy. i.e. if you did the upgrade manually.

This is a guide for moving from Courier to Deploy. 
The first step is to acquire the latest Umbraco Deploy package from [http://nightly.umbraco.org/?container=umbraco-deploy-release](http://nightly.umbraco.org/?container=umbraco-deploy-release)

For a version 7.x website, choose the latest "UmbracoDeploy.v2.x.x.zip" file.

> Next, request a valid Deploy Key from the support heroes. Use the in-app chat in the Umbraco Cloud portal to get this.

Reviewer: ```suggestion
Next, request a valid Deploy Key from Umbraco Support. This can be done through the in-app chat in the Umbraco Cloud Portal or by reaching out to contact@umbraco.com.
```

Removed "support heroes", as this might become a bit internal.
Also provided one more option for getting in touch with our support.

Author: Great improvement :-)

* /bin/Concorde.CacheHandler.dll
* /bin/Concorde.CacheHandler.LiveEdit.dll
* /bin/Concorde.CacheHandler.UI.dll
* /bin/Our.Umbraco.Courier.DataResolvers.dll
* /bin/Umbraco.Courier.Contrib.Resolvers.dll
* /bin/Umbraco.Courier.Core.dll
* /bin/Umbraco.Courier.DataResolvers.dll
* /bin/Umbraco.Courier.EventHandlers.V7.dll
* /bin/Umbraco.Courier.FormsProvider.dll
* /bin/Umbraco.Courier.Persistence.dll
* /bin/Umbraco.Courier.Persistence.V7.NHibernate.dll
* /bin/Umbraco.Courier.Providers.dll
* /bin/Umbraco.Courier.RepositoryProviders.dll
* /config/courier.config
* /config/splashes/noNodes.aspx
* /umbraco/Plugins/Courier/Webservices/Repository.asmx

Also, these 3rd party dlls have been part of Courier. If you are not using them in your custom code, you should ensure that they get deleted as well.

* /bin/Antlr3.Runtime.dll
* /bin/Castle.Core.dll
* /bin/FluentNHibernate.dll
* /bin/lesi.Collections.dll
* /bin/NHibernate.dll

If you have a NuGet reference to "Umbraco.Courier.Contrib", you should uninstall that and delete the related .dll from `bin` as well.

Finally all `.courier` files found in `/data/Revision` needs to be deleted as well, as they will no longer be used. 

> **NOTE:** Do not remove the `/data/Revision` folder itself, only the folders inside the `/data/Revision` folder - you will need this folder for Deploy as well.

Reviewer: ```suggestion
:::note
Do not remove the `/data/Revision` folder itself, only the folders inside the `/data/Revision` folder - you will need this folder for Deploy as well.
:::
```

Putting the note into the "note format"

## 2 Adding Umbraco Deploy

Adding Umbraco Deploy will be simple, all content from the ZIP received (except `/config/UmbracoDeploy.config`) needs to be copied into the webroot.

> **NOTE:** Do not overwrite the existing file `/config/UmbracoDeploy.config` with the empty one from the ZIP. It already has the correct settings for your site.

Reviewer: ```suggestion
:::note
Do not overwrite the existing `/config/UmbracoDeploy.config` file with the empty one from the ZIP. It already has the correct settings for your site.
:::
```

Author: Oh, I didn't know about the "note format" - cool :-)
